### ISSUE TITLE
AppStream metadata

### ISSUE THREAD

**Issue by korewaChino** [terrapkg/packages#1327]
We should export AppStream metadata for Terra for some if not most packages for easier graphical installation using PackageKit clients like GNOME Software or KDE Discover.

My recommendations for initial packages are:
- `xpadneo`
- Prism Launcher
- Armcord
- Black Box
- VOICEVOX
- Authy

**Comment by ThatBlockyPenguin** [comment]
Hi! Is there any progress on this? I'm quite new to Fedora's ecosystem, but I'd be willing to help in any way that I can?

**Comment by korewaChino** [comment]
actually this issue should be completed, closing this

**Comment by ThatBlockyPenguin** [comment]
Hm, that's odd, as I still am unable to browse Terra packages from GNOME Software? Should I open a new issue for this?

**Comment by lleyton** [comment]
no worries, I'll reopen this issue, by chance, does `pkcon refresh force` make a difference?

**Comment by ThatBlockyPenguin** [comment]
It does seem to have helped, but not with everything...

Ghostty now shows up, whereas it didn't before:

<img src="https://github.com/user-attachments/assets/17f77784-f8bc-4ad6-b407-efef5101b5e7" />

However the specific Minecraft Launcher that I have installed from Terra doesn't:

<video src="https://github.com/user-attachments/assets/349c4c71-0586-4ed6-b211-03745abf774f"></video>

In fact, it doesn't even show in GNOME Software's "Installed" page

**Comment by korewaChino** [comment]
The Minecraft Java launcher doesn't have AppStream metadata, so yeah

**Comment by ThatBlockyPenguin** [comment]
Genuine question - is it not possible to have AppStream metadata for everything? Like, what's stopping certain programs (i.e. the Minecraft Launcher) from having an entry?

Could this process not be automated?

**Comment by lleyton** [comment]
TL;DR: The package needs to ship an AppStream XML file. It could be generated, but it would be extremely barebones.

**Comment by ThatBlockyPenguin** [comment]
Gotcha, OK. So are the packages mentioned in #6882 the only ones in Terra that don't have metadata?

IMO, a barebones entry is better than none at all, as if you're coming from Windows/Mac and told "go look in this app store for all your software", but not everything that is available shows up, then it can feel quite frustrating. Not sure what Fyra's official stance here is?

**Comment by korewaChino** [comment]
> Gotcha, OK. So are the packages mentioned in [#6882](https://github.com/terrapkg/packages/issues/6882) the only ones in Terra that don't have metadata?

Sorry, that list isn't complete yet. I would need to complete the list someday
